lightning-background-processor/src/lib.rs

```diff
@@ -1189,19 +1191,22 @@ mod tests {
 
 	#[test]
 	fn test_timer_tick_called() {
-		// Test that ChannelManager's and PeerManager's `timer_tick_occurred` is called every
-		// `FRESHNESS_TIMER`.
+		// Test that `ChannelManager::timer_tick_occurred` and
+		// `ChainMonitor::rebroadcast_pending_claims` is called every `FRESHNESS_TIMER` and
```

Comment:
`FRESHNESS_TIMER=60`, should we introduce a bit of noise by randomnizing rebroadcast interval to avoid leaking a static rebroadcast pattern to a deanonymization attacker at the transaction-propagation layer ? independently of the logic that the transaction-propagation layer can have